src/pkg/backup/details/details_test.go

```diff
@@ -207,7 +234,7 @@ func (suite *DetailsUnitSuite) TestDetailsModel_Path() {
 					Entries: test.ents,
 				},
 			}
-			assert.Equal(t, test.expectRefs, d.Paths())
+			assert.Equal(t, test.expectRepoRefs, d.Paths())
```

Comment:
nit: we probably want `ElementsMatch()`?
```suggestion
			assert.ElementsMatch(t, test.expectRepoRefs, d.Paths())
```

Reply:
always gets me.